src/Extensions/Extensions.csproj

```diff
@@ -24,7 +24,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.18" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.41" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.2.0" />
```

Comment:
Version 2.2.0 is deprecated - https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection/2.2.0.

We can upgrade to version 2.1.1 - https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection/2.1.1